addons/microsoft_calendar/views/res_users_views.xml

```diff
@@ -6,20 +6,23 @@
             <field name="model">res.users</field>
             <field name="inherit_id" ref="base.view_users_form"/>
             <field name="arch" type="xml">
-                <notebook colspan="4" position="inside">
-                    <page string="Calendar" name="calendar" groups="base.group_system">
+                <xpath expr="//page[@name='calendar']" position='attributes'>
+                    <attribute name="invisible">0</attribute>
+                </xpath>
+                <group name="calendar_accounts" position="inside">
+                    <group string="Office 365 Calendar">
                         <group>
-                            <field name="microsoft_calendar_rtoken" readonly="1"/>
-                            <field name="microsoft_calendar_token" readonly="1"/>
-                            <field name="microsoft_calendar_token_validity" readonly="1"/>
-                            <field name="microsoft_calendar_sync_token" readonly="1"/>
+                            <field name="microsoft_calendar_rtoken" string="Refresh Token" readonly="1"/>
+                            <field name="microsoft_calendar_token" string="User Token" readonly="1"/>
+                            <field name="microsoft_calendar_token_validity" string="Token Validity" readonly="1"/>
+                            <field name="microsoft_calendar_sync_token" string="Next Sync Token" readonly="1"/>
```

Comment:
Also, about this part, Google fields do not have "Google" in front of every label.
I would align Microsoft with that and change the actual fields definition instead of overriding the strings here.

Reply:
As both the fields are on the same object, it will give warning if we add same labels for multiple fields.
Logs:-
2020-09-02 06:21:35,483 24794 WARNING test odoo.addons.base.models.ir_model: Two fields (google_calendar_rtoken, microsoft_calendar_rtoken) of res.users() have the same label: Refresh Token. 
2020-09-02 06:21:35,483 24794 WARNING test odoo.addons.base.models.ir_model: Two fields (google_calendar_token, microsoft_calendar_token) of res.users() have the same label: User token. 
2020-09-02 06:21:35,483 24794 WARNING test odoo.addons.base.models.ir_model: Two fields (google_calendar_token_validity, microsoft_calendar_token_validity) of res.users() have the same label: Token Validity.

Reply:
You're right, didn't think about that.
You can leave it as is then.